packages/common/src/cloudwatch-event.ts

```diff
@@ -0,0 +1,88 @@
+import * as log from './log';
+import { isOneOf } from './util';
+import { AwsCloudWatchEvent } from './types';
+
+/**
+ * Determine if Cloudwatch event is a Step Function event
+ *
+ * @param {AwsCloudWatchEvent} event - A Cloudwatch event
+ * @returns {boolean} True if event is a Step Function event
+ */
+export const isSfExecutionEvent = (event: AwsCloudWatchEvent) =>
+  event.source === 'aws.states';
+
+/**
+ * Determine if Step Function is in a terminal state.
+ *
+ * @param {string} status - A Step Function execution status from a Cloudwatch event
+ * @returns {boolean} True if Step Function is in terminal state.
+ */
+export const isTerminalSfStatus: (status: string) => boolean = isOneOf([
```

Comment:
Oh, is this the right syntax for specifying the return type? I thought it was:

`export const isTerminalSfStatus = (status: string): boolean =>`

Is this because of how the function just calls `isOneOf` directly?

Reply:
This isn't actually specifying the return type of `isTerminalSfStatus`. It's explicitly setting the type of `isTerminalSfStatus`. `isOneOf()` returns a function, which is being assigned to `isTerminalSfStatus`. This is one of those few cases where TS is unable to properly determine the return type of `isOneOf`, so I'm explicitly saying what `isOneOf` returns.